### PR TITLE
Restore ability to set Speaker to transcripts

### PR DESCRIPTION
**Pull request by mdmohsin7**
Part of #1230 

- [x] Assign segments to people
- [x] Add people
- [x] ~Authorize and save conversation recordings~
- [x] ~Assign people and update their speech samples~

https://github.com/user-attachments/assets/c1a99a97-1a13-4fe2-86c0-70607fdf3425

https://github.com/user-attachments/assets/9add1b5d-8479-41ea-ad4a-62bb4bd11c87

### CODE REVIEW

**Comment by beastoin** [comment]
Screenshot 2024-12-18 at 09 04 28

cool man, btw let's push 2 these changes to our users then see how they use it. 

we don't really need the automation for now.

i also need your research on how another apps are doing it.

@mdmohsin7

**Comment by mdmohsin7** [comment]
Alright then, let me comment the recording functionality for now. I'll also improve the tagging ui a bit to make it easy for users to know that they can assign segments to people

> i also need your research on how another apps are doing it.

Alright will try to find out

@beastoin

File: backend/routers/transcribe_v2.py [hunk 1]
```diff
@@ -470,17 +503,12 @@ async def receive_audio(dg_socket1, dg_socket2, soniox_socket, speechmatics_sock
...
             while websocket_active:
                 data = await websocket.receive_bytes()
-                # save the data to a file
-                # data_length = len(data)
-                # f.write(struct.pack('I', data_length))  # Write length as 4 bytes
-                # f.write(data)
-
                 if codec == 'opus' and sample_rate == 16000:
                     data = decoder.decode(bytes(data), frame_size=160)
+                    audio_data.extend(data)
```
Comment (beastoin) [inline backend/routers/transcribe_v2.py]: comment this line pls sir @mdmohsin7

Comment (mdmohsin7) [reply backend/routers/transcribe_v2.py]: my bad missed it, let me

Comment (mdmohsin7) [reply backend/routers/transcribe_v2.py]: Done @beastoin

**Comment by beastoin** [comment]
lgtm @mdmohsin7 🥳

**Pull request merged**